content/tutorials/token-authentication.textile

```diff
@@ -607,7 +607,7 @@ blang[android].
   ...
   dependencies {
       ...
-      compile 'io.ably:ably-android:0.8.7'
+      compile 'io.ably:ably-android:1.0.0'
```

Comment:
I must admit @paddybyers I really don't see why we're special casing Java over every other library?  The developer referencing the Java lib should decide if they want to lock to a specific version or not, however the documentation itself, unless we update it for every Java release, will simply encourage people to use old unpatched versions of the library. My vote is to add a note to say it's up to the developer to decide if they want to lock into a major, minor or patch release, but the docs should encourage a user to lock into the version that matches this documentation which is `1.0+`